mesop/pip_package/requirements.txt

```diff
@@ -1,9 +1,7 @@
 absl-py
-deepdiff==6.7.1
+deepdiff
```

Comment:
[question] is there a reason to remove the version here? Mainly pinning to downstream. Right now, the latest is 7.0.1 which I'm worried could have breaking changes.

Reply:
i'll undo this. since there's already a lot of changes in this PR. I think it's generally not a good thing to pin to an exact version in Python because then people will run into version dependency conflicts and not be able to install mesop in their python environment (the ideal solution is to use virt)